docs/index.md

```diff
@@ -11,29 +11,29 @@ There are four types of use cases for Node generator.
 
 #### (1) Connection to cloud services
 The http request node can easily connect to cloud services via REST API.
-But in terms of marketing, instead of http request node, the original node for the cloud services will be effective to increase the number of users because details of node properties, node information and documents will contribute to the usability of the node.
-In general, cloud service providers publish Open API document with REST API for users to test it on Swagger UI.
-Node generator uses this Open API document to create nodes for connection to the cloud service.
+From a marketing perspective, instead of using the http request node, an original node for a cloud service is more effective to increase the number of users because having details of node properties, node information and documents can help contribute to the usability of the node.
```

Comment:
maybe instead - 'From a marketing perspective, having a dedicated node for a cloud service is more attractive to users because having preset node properties, and targeted information and documents can help improve the usability of the node.